aptos-move/vm-genesis/src/lib.rs

```diff
@@ -53,6 +53,8 @@ const MICRO_SECONDS_PER_SECOND: u64 = 1_000_000;
 pub struct GenesisConfiguration {
     pub allow_new_validators: bool,
     pub epoch_duration_secs: u64,
+    // If true, genesis will create a special core resources account that can mint coins.
+    pub is_test: bool,
```

Comment:
Newly introduced config that decides whether core resources account should be initialized or not.